ext/opentelemetry-ext-requests/tests/test_requests_integration.py

```diff
@@ -183,6 +185,45 @@ def test_distributed_context(self):
         finally:
             propagators.set_global_httptextformat(previous_propagator)
 
+    def test_apply_custom_attributes_on_span(self):
+        resource = resources.Resource.create({})
```

Comment:
This `resource` is only meaningful for the `test_custom_tracer_provider` case, other cases could avoid it (also remove check on line 225).

Reply:
Done.